### PR TITLE
[v5] Make `system` accessible in all custom actions

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: packages/core/test/system.test.ts [hunk 1]
```diff
@@ -127,4 +127,39 @@ describe('system', () => {
...
+      },
+      entry: (_, __, { system }) => {
+        expect(system!.get('test')).toBeDefined();
+      }
+    });
+
+    interpret(machine).start();
+  });
+
+  it('should be accessible in referenced custom actions', () => {
```
Comment (Andarist) [inline packages/core/test/system.test.ts:145]: I only fixed this for "custom actions" aka functions. Previously it was only provided to inline ones but references ones didn't have that privilege. 

It's still not accessible to `assign`, `sendTo` (!), and more. Do we want to add it to every builtin action type?

Comment (davidkpiano) [reply packages/core/test/system.test.ts:145]: I'd say so 👍

**Review by davidkpiano — APPROVED** [review]
Approved, but feel free to add it to the other built-in actions

File: packages/core/test/system.test.ts (hunk 1 above)
Comment (davidkpiano) [reply packages/core/test/system.test.ts:147]: Added here: 4adcba4f648042f249269eb3441c88ccb4836136

**Pull request merged**